handbook/engineering/distribution/recurring_processes.md

```diff
@@ -0,0 +1,146 @@
+# Recurring processes
```

Comment:
@slimsag I documented the recurring processes on the Distribution team, including milestone planning and OKR setting, which the project lead (i.e., you) is responsible for. PTAL and let me know if this accurately reflects our process as you understand it and whether I can clarify anything.

Reply:
Some of this content probably can/should be upleveled to be relevant to all eng teams, but I am fine merging this as-is for now I can can work to standardize across teams independently.